rubicon_ml/client/project.py

```diff
@@ -256,11 +274,13 @@ def _filter_experiments(self, experiments, tags, qtype):
                 for e in experiments
                 if has_tag_requirements(e.tags, tags, qtype)
             ]
+            if name is not None:
```

Comment:
this is only gonna filter by name if there are also tags given to filter by. it should be able to do both individually or together